api/scpca_portal/readme_file.py

```diff
@@ -12,6 +14,112 @@
 TEMPLATE_ROOT = settings.TEMPLATE_PATH / "readme"
 TEMPLATE_FILE_PATH = TEMPLATE_ROOT / "readme.md"
 
+# Dataset Readme Templates
+README_ROOT = settings.TEMPLATE_PATH / "dataset_readme"
+HEADER_TEMPLATE = README_ROOT / "header/index.md"
+USAGE_TEMPLATE = README_ROOT / "usage/index.md"
+CHANGELOG_TEMPLATE = README_ROOT / "changelog/index.md"
+CONTACT_TEMPLATE = README_ROOT / "contact/index.md"
+CITATION_TEMPLATE = README_ROOT / "citation/index.md"
+TERMS_TEMPLATE = README_ROOT / "terms_of_use/index.md"
+
+
+# Contents
+DATASET_CONTENTS_ROOT = README_ROOT / "contents"
+
+
+def merge_partials(partials: list[str]):
+    """
+    Takes a list of rendered templates, strips, removes empty and combines with new line.
+    """
+    # For the conte
```

Comment:
```suggestion
```

As we covered in our 1:1,  could we perhaps remove this comment? The docs string explains the method very well, and the line itself is self-explanatory.